src/agent/agentLoader.ts

```diff
@@ -69,6 +69,9 @@ export class AgentLoader {
                     redis: {
                         enabled: true
                     },
+                    bunyan: {
```

Comment:
Looks like you are missing winston here, right? default should be false for all other agents, true only for AKS

Reply:
Removed here and added to where winston is also set in the AKS loader. New functionality should fall into 3 categories that I've tested:
1. Non-AKS auto attach - all instrumentations that aren't logging are enabled, but console, winston, and bunyan are disabled.
2. AKS auto attach - instrumentations and logging is enabled by default.
3. Manual instrumentation - all instrumentations, bunyan, and winston are enabled by default, but console is disabled.